src/generated/graphql/types.ts

```diff
@@ -26,6 +26,21 @@ export type Scalars = {
   Url: any;
 };
 
+/**
```

Comment:
These changes all arrived here as part of the codegen step of the build. I _assume_ that these are derived from some sort of introspection on the graph. They neither look harmless, nor do they seem directly relevant to the intent of this PR; if folks with better understanding of the graph semantics can advise, I'd appreciate. Happy to punt on this PR is this is questionable.

Reply:
These appear to be updates to the graph made by the Pocket team recently. They're all additive changes, too, which means they won't affect anything this proxy relies on.